chromium_src/BUILD.gn

```diff
@@ -4,14 +4,32 @@
 
 import("//build/config/ozone.gni")
 import("//build/config/ui.gni")
+import("//chrome/browser/buildflags.gni")
 import("//components/spellcheck/spellcheck_build_features.gni")
 import("//electron/buildflags/buildflags.gni")
 import("//printing/buildflags/buildflags.gni")
 import("//third_party/widevine/cdm/widevine.gni")
 
+copy("shaded_resources_copy") {
+  deps = [ "//electron:resources" ]
+  sources = [ "$root_gen_dir/electron/grit/electron_resources.h" ]
+  outputs = [ "$target_gen_dir/chrome/grit/generated_resources.h" ]
```

Comment:
`patches/chromium/fix_add_support_for_webauthn_on_macos.patch ` already uses `electron/grit/electron_resources.h` where needed, why is this step necessary to copy contents to `chrome/grit/generated_resources.h` ?

Reply:
Good question, I'll see if I can remove it.

Reply:
@deepak1556 Removing that code worked fine, I've pushed a fix for that.